deploy.sh

```diff
@@ -37,47 +37,49 @@ function is_working_tree_dirty {
 # for gh-pages, use unpkg.com for dependencies
 function replace_onboarding_src {
   local local_src="node_modules/@metamask/onboarding/dist/metamask-onboarding.bundle.js"
-  local web_src="unpkg.com/metamask-onboarding"
+  local web_src="https://unpkg.com/@metamask/onboarding"
   local target_file="index.html"
 
-  sed -i "" -e "s/${local_src}/${web_src}/g" "${target_file}"
+  sed -i "" -e "s#$local_src#$web_src#g" "$target_file"
```

Comment:
Was it necessary to remove all of the curly brackets? The unbound variable error you mentioned seeing on Slack may have been specific to the `@` variable - generally speaking the curly braces are good, as they remove the possibility of ambiguity (e.g. it might end up substituting a variable named `web` here instead of `web_src`, if one existed)

Reply:
Ah, so they're not completely unnecessary? ;)

Added back

Reply:
Hah, I mean they're unnecessary for the case of `${@}`. There are no other characters in that string that could be confused as part of the variable name.

It's an old habit of mine that I don't see that often in the wild, and is admittedly a bit silly. I just started using braces for all variables at some point.